its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/CppTest.java

```diff
@@ -121,11 +122,10 @@ private static TempDirectory getBuildWrapperDir(AnalysisContext context) throws
   }
 
   private static void runMSBuildWithBuildWrapper(Path projectDir, File buildWrapperPath, File outDir, String... arguments) {
-    Path msBuildPath = TestUtils.getMsBuildPath(ORCHESTRATOR);
     new GeneralCommand(buildWrapperPath.toString(), projectDir)
       .addArgument("--out-dir")
       .addArgument(outDir.toString())
-      .addArgument(msBuildPath.toString())
+      .addArgument(BuildCommand.msBuildPath())
```

Comment:
Migrate to the version from BuildCommand that does not need orchestrator. `TestUtils.getMsBuildPath` is gone.